lib/Zonemaster/Backend/Config.pm

```diff
@@ -185,32 +185,88 @@ sub PollingInterval {
     return $self->{cfg}->val( 'DB', 'polling_interval' );
 }
 
+
+=head2 MaxZonemasterExecutionTime
+
+=head3 INPUT
+
+'max_zonemaster_execution_time' from [ZONEMASTER] section in ini file. See
+L<https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#max_zonemaster_execution_time>.
+
+=head3 RETURNS
+
+Integer (number of seconds)
```

Comment:
It is out of scope for this PR to add validation for anything execept `age_reuse_previous_test`, but that is a good suggestion. For `age_reuse_previous_test` the validation is done such as if the value is not numeric and positive, it will be ignored. Should it croak instead?

I do not understand how to used `load_config` method for validation.

Reply:
On second thought it's probably better to leave the validation out for now and deal with it in #685.